BraintreeDropIn/BTCardFormViewController.m

```diff
@@ -408,13 +416,18 @@ - (void)setCollapsed:(BOOL)collapsed {
             self.mobileCountryCodeField.hidden = ![self.requiredFields containsObject:self.mobileCountryCodeField] || collapsed;
             self.mobilePhoneField.hidden = ![self.requiredFields containsObject:self.mobilePhoneField] || collapsed;
             self.enrollmentFooter.hidden = self.mobilePhoneField.hidden;
+            self.shouldVaultCardSwitchField.hidden = ![self shouldDisplaySaveCardToggle] || collapsed;
             
             [self updateFormBorders];
             [self updateSubmitButton];
         }];
     });
 }
 
+- (BOOL)shouldDisplaySaveCardToggle {
+    return self.dropInRequest.allowVaultCardOverride && self.apiClient.tokenizationKey == nil;
```

Comment:
To confirm, vaulting cannot occur without a `client_token` and cannot occur with a `tokenization_key` right?